packages/react-components/src/listbox/docs/Avatar.jsx

```diff
@@ -6,7 +6,7 @@ export function SmallAvatar({ children, ...rest }) {
             {...mergeProps(
                 rest,
                 {
-                    className: "w5 h5 f8 br-100 flex items-center justify-center"
+                    className: "w5 h5 f9 bg-primary-500 white br-100 flex items-center justify-center"
```

Comment:
--o-ui-alias-background-primary-1 should be used because of dark mode

Reply:
Would love to be I think https://github.com/gsoft-inc/sg-orbit/pull/472 deleted the tachyons class?